src/System.Private.ServiceModel/src/Extensions/ReflectionExtensions.cs

```diff
@@ -92,7 +92,7 @@ public static bool IsValueType(this Type type)
         }
         public static InterfaceMapping GetInterfaceMap(this Type type, Type interfaceType)
         {
-            throw ExceptionHelper.PlatformNotSupported();
+            return type.GetTypeInfo().GetRuntimeInterfaceMap(interfaceType);
```

Comment:
This is a one line product fix needed to get the test to pass. For pragmatic reasons, it's worth making that one change rather than check in the test, have the test fail, open a new issue, then submit a separate pull request just for this one line.